CK3toEU4/Data_Files/blankMod/output/common/on_actions/ZZZ_added_on_actions.txt

```diff
@@ -1,220 +1,222 @@
-						OR = {
-							religion = tengri_pagan_reformed
-							ruler_religion = tengri_pagan_reformed
-							heir_religion = tengri_pagan_reformed
-							consort_religion = tengri_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = tengri_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_tengri_pagan_reformed
-		}
-		#West African
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = west_african_pagan_reformed
-							ruler_religion = west_african_pagan_reformed
-							heir_religion = west_african_pagan_reformed
-							consort_religion = west_african_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = west_african_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_west_african_pagan_reformed
-		}
-		#Hellenic
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = hellenic_pagan_reformed
-							ruler_religion = hellenic_pagan_reformed
-							heir_religion = hellenic_pagan_reformed
-							consort_religion = hellenic_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = hellenic_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_hellenic_pagan_reformed
-		}
-		#Zun
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = zun_pagan_reformed
-							ruler_religion = zun_pagan_reformed
-							heir_religion = zun_pagan_reformed
-							consort_religion = zun_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = zun_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_zun_pagan_reformed
-		}
-		#Bon
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = bon_reformed
-							ruler_religion = bon_reformed
-							heir_religion = bon_reformed
-							consort_religion = bon_reformed
-						}
-					}
-					any_province = {
-						religion = bon_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_bon_reformed
-		}
-		#Kushite
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = kushitism_pagan_reformed
-							ruler_religion = kushitism_pagan_reformed
-							heir_religion = kushitism_pagan_reformed
-							consort_religion = kushitism_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = kushitism_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_kushitism_pagan_reformed
-		}
-		#Magyar
-		if = {
-			limit = {
-				OR = {
-					any_country = {
-						OR = {
-							religion = magyar_pagan_reformed
-							ruler_religion = magyar_pagan_reformed
-							heir_religion = magyar_pagan_reformed
-							consort_religion = magyar_pagan_reformed
-						}
-					}
-					any_province = {
-						religion = magyar_pagan_reformed
-					}
-				}
-			}
-			set_global_flag = ZZZ_enhanced_magyar_pagan_reformed
-		}
-	}
-}
+###Not Needed for CK3 since all reformed faiths are dynamic
```

Comment:
Why not delete this entire block if not needed?

Reply:
Just in case I end up needing it again. It's all comments so it doesn't affect anything. If you really want me to, I will delete it.

Reply:
Nah, we'll drop it if you don't use it in some time.